src/platforms/android/include.gradle

```diff
@@ -25,20 +21,16 @@ repositories {
 
 dependencies {
     def googlePlayServicesVersion = project.hasProperty('googlePlayServicesVersion') ? project.googlePlayServicesVersion : '10.2.6'
-    compile "com.google.android.gms:play-services-auth:$googlePlayServicesVersion"
+    implementation "com.google.android.gms:play-services-auth:$googlePlayServicesVersion"
 
     def facebookSDKVersion = project.hasProperty('facebookSDKVersion') ? project.facebookSDKVersion : '[4, 5)'
     // https://github.com/mkloubert/nativescript-social-login/issues/32#issue-285587592
-    compile ("com.facebook.android:facebook-login:$facebookSDKVersion"){
+    implementation ("com.facebook.android:facebook-login:[4,5)"){

```

Comment:
@SalehMahmood is there a particular need for removing `$facebookSDKVersion`?

Reply:
No, not quite

Reply:
👍